scripts/package.sh

```diff
@@ -15,7 +15,7 @@ for f in $(find "${SOURCE_DIR}/sql/updates/" -name "*.sql"); do
     cp $f ${BUILD_DIR}/${PACKAGE_NAME}/src/${dest_filename}
 done
 
-cp ${SOURCE_DIR}/lanterndb.control ${BUILD_DIR}/${PACKAGE_NAME}/src
+cp ${BUILD_DIR}/lanterndb.control ${BUILD_DIR}/${PACKAGE_NAME}/src
```

Comment:
What does this change do? What changed here?

Reply:
This was changed due to this PR https://github.com/lanterndata/lanterndb/pull/53

Reply:
got it, thanks!